DESCRIPTION

```diff
@@ -23,7 +23,7 @@ License: file LICENSE
 Encoding: UTF-8
 LazyData: true
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.1.1
+RoxygenNote: 7.1.2
```

Comment:
Happens automatically when I redocument with updated packages. If you're behind you may run:

```r
update.packages(ask = FALSE)
```